test/README.md

```diff
@@ -0,0 +1,22 @@
+Scylla in-source tests.
+
+For details on how to run the tests, see docs/dev/testing.md
+
+Shared C++ utils, libraries are in lib/, for Pythoin - pylib/
+
+alternator - client/server tests for Scylla Alternator
```

Comment:
I don't know what "client/server tests" mean here, these tests test a lot more than just the server. It tests the entire DynamoDB API and all its various functionalities.
I suggest that you use the same description as you did for cql-pytest - e.g., "Python tests which connect to a single server and use the DynamoDB API".

Reply:
Done.